api/config/config.js

```diff
@@ -26,5 +26,14 @@ module.exports = {
 
   // weak:
   // Unrestricted passwords allowed
-  passwordStrength: 'weak'
+  passwordStrength: 'weak',
```

Comment:
is this file supposed to be here?

Reply:
No, the config file should not be committed - and is now added to gitignore. I did not remove it from GitHub during commits because I don’t want to disrupt dev on other people’s accounts since this file is relied upon. When deploying we can generate a new hash and mailer keys + token when removing this file from public access

Reply:
lets add the to the gitignore, the contributions right now are mostly me and thai so it shouldnt be too disruptive.

Reply:
ok